scripts/migrations/002-remove-webview-flags.test.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const { platform } = require('os');
+
+const input = JSON.stringify({
+  "test": {
+    "__compat": {
+      "support": {
+        "webview_android": {
```

Comment:
A suggestion here: make another test case (or expand this one with an array) showing a webview entry that would _not_ be modified with the migration?